pyspecdata/core.py

```diff
@@ -158,6 +159,8 @@ def mydiff(data,axis = -1):
 def normal_attrs(obj):
     myattrs = [x for x in dir(obj) if not ismethod(obj.__getattribute__(x))]
     myattrs = [x for x in myattrs if not x[0:2] == '__']
+    # next line filters out properties
+    myattrs = [x for x in myattrs if x not in ['C','angle','imag','real']]
```

Comment:
I think the line currently in master is more up-to-date -- try `git diff master core`, and jump to line 163 in your current version to check it out.

I _think_ you should see there's a list defined that lists all the attributes... is so, that is the more up to date version

Reply:
although, looking below it's possible this is the simpler version, since I notice we just re-use normal_attrs several times

Reply:
The difference between core here and core in master is that you have removed lines 162 and 163 here, but nothing was added in its place

Reply:
actually, I don't see that difference now -- maybe pull down the most recent version of master?